src/menus/FileMenus.cpp

```diff
@@ -34,6 +34,35 @@
 // private helper classes and functions
 namespace {
 
+bool ExportWithPrompt(AudacityProject &project,
+                      Exporter &exporter,
+                      const FileExtension& format,
+                      double t0, double t1, bool selectedOnly)
+{
```

Comment:
This new function shows the file dialog after the setting of export range, but previous to this commit, the two things were in the opposite order (as they remain for timer recording).

So there may be an unintended, though unimportant, change in behavior.  Such as giving the warning dialog before any file selection.

Better to change no behavior unless really meaning to.

Reply:
The behavior is restored in one of the latest commits